qiskit/chemistry/qubit_transformations/fermionic_transformation.py

```diff
@@ -0,0 +1,566 @@
+        """
+        # Finding all the symmetries using the find_Z2_symmetries:
+        taper_coef = []
+        for sym in z2_symmetries.symmetries:
+            # pylint: disable=no-member
+            coef = -1 if np.logical_xor.reduce(np.logical_and(sym.z[::-1], hf_str)) else 1
+            taper_coef.append(coef)
+        z2_symmetries.tapering_values = taper_coef
+        return z2_symmetries
+
+    # pylint: disable=unused-argument
+    def interpret(self,
+                  eigenvalue: float, eigenstate: List[float], aux_values: list
+                  ) -> MolecularGroundStateResult:
+
+        """Interpret eigenvalue and eigenstate of qubit Hamiltonian w.r.t. driver.
+
+        Args:
+            TODO
+
+        Returns:
+            GroundState Result TODO
+        """
+
+        mgsr = MolecularGroundStateResult()
+        mgsr.hartree_fock_energy = self._hf_energy
+        mgsr.nuclear_repulsion_energy = self._nuclear_repulsion_energy
+        if self._nuclear_dipole_moment is not None:
+            mgsr.nuclear_dipole_moment = tuple(x for x in self._nuclear_dipole_moment)
+        mgsr.computed_electronic_energy = eigenvalue.real
+        mgsr.ph_extracted_energy = self._ph_energy_shift
+        mgsr.frozen_extracted_energy = self._energy_shift
+        aux_ops_vals = aux_values
+        if aux_ops_vals is not None:
+            # Dipole results if dipole aux ops were present
+            dipole_idx = 3
+            if len(aux_ops_vals) > dipole_idx:
+                mgsr.reverse_dipole_sign = self._reverse_dipole_sign
+                dipm = []
+                for i in range(dipole_idx, dipole_idx + 3):  # Gets X, Y and Z components
+                    dipm.append(aux_ops_vals[i][0].real if aux_ops_vals[i] is not None else None)
+                mgsr.computed_dipole_moment = cast(DipoleTuple, tuple(dipm))
+                mgsr.ph_extracted_dipole_moment = (self._ph_x_dipole_shift,
+                                                   self._ph_y_dipole_shift,
+                                                   self._ph_z_dipole_shift)
+                mgsr.frozen_extracted_dipole_moment = (self._x_dipole_shift,
+                                                       self._y_dipole_shift,
+                                                       self._z_dipole_shift)
+            # The first 3 entries are num particles, total angular momentum and magnetization
+            mgsr.num_particles = aux_ops_vals[0][0].real \
+                if aux_ops_vals[0] is not None else None
+            mgsr.total_angular_momentum = aux_ops_vals[1][0].real \
+                if aux_ops_vals[1] is not None else None
+            mgsr.magnetization = aux_ops_vals[2][0].real \
+                if aux_ops_vals[2] is not None else None
+        return mgsr
+
+    # Called by public superclass method process_algorithm_result to complete specific processing
+    def _process_algorithm_result(self, algo_result: MinimumEigensolverResult) \
```

Comment:
This transformation will end up getting used with either MinimumEigenSolver or Eigensolver algorithms and interpreting the results is different (as was done before in the Hamiltonian)